tests.py

```diff
@@ -258,9 +285,21 @@ def test_subclauses_arithmetic_booleans(self):
                 expected_output.strip() + '\n', False)
 
 if __name__ == '__main__':
+    # NOTE: intentionally skipping test_simple0_log_replay for the set of IPC
+    # tests because it causes other test cases to block indefinitely on ZeroMQ
+    # IPC
+    IPC_TESTS_SKIP = ['test_simple0_log_replay']
```

Comment:
It's kind of indefinitely-temporary. Log replay over IPC doesn't seem to be a key use case and if the log replay test runs with the IPC module, the full test suite hangs.

At any rate, I found a way to skip this test just when it's run with an IPC module. It required adding a teardown function and cleanly closing the ZeroMQ socket (which we should be doing anyhow to prevent other test failures from making the test suite hang).